package.json

```diff
@@ -13,8 +13,8 @@
   ],
   "icon": "images/icon.png",
   "galleryBanner": {
-    "color": "#5c2d91",
```

Comment:
Do you meant to change the banner color?

Reply:
Yes. This commit is suggesting a new icon and banner color. If you feel the design needs more work (it was put together quickly), feel free to discard it and let me know if you would like to bounce other ideas.